src/geom/SkyWcs.cc

```diff
@@ -0,0 +1,249 @@
+/*
+ * LSST Data Management System
+ * See COPYRIGHT file at the top of the source tree.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include <exception>
+#include <memory>
+#include <ostream>
+#include <sstream>
+#include <vector>
+
+#include "astshim.h"
+
+#include "lsst/afw/formatters/Utils.h"
+#include "lsst/afw/geom/detail/frameSetUtils.h"
+#include "lsst/afw/geom/Point.h"
+#include "lsst/afw/geom/SpherePoint.h"
+#include "lsst/afw/geom/SkyWcs.h"
+#include "lsst/daf/base/PropertyList.h"
+#include "lsst/pex/exceptions.h"
+
+namespace lsst {
+namespace afw {
+namespace geom {
+namespace {
+
+inline double square(double x) { return x * x; }
+
+}  // namespace
+
+Eigen::Matrix2d makeCdMatrix(Angle const& scale, Angle const& orientation, bool flipX) {
+    Eigen::Matrix2d cdMatrix;
+    double orientRad = orientation.asRadians();
+    double scaleDeg = scale.asDegrees();
+    double xmult = flipX ? -1.0 : 1.0;
+    cdMatrix(0, 0) = std::cos(orientRad) * scaleDeg * xmult;
+    cdMatrix(0, 1) = std::sin(orientRad) * scaleDeg;
+    cdMatrix(1, 0) = -std::sin(orientRad) * scaleDeg * xmult;
+    cdMatrix(1, 1) = std::cos(orientRad) * scaleDeg;
```

Comment:
As I've said, it doesn't really matter here, but I was curious so I investigated.  The answer: gcc will, clang will not.
More details at http://stackoverflow.com/a/37877352/834250
Verified myself using https://godbolt.org .